src/sid/statistics.py

```diff
@@ -78,10 +78,10 @@ def calculate_r_zero(df: pd.DataFrame, window_length: int = 7) -> pd.Series:
 
     grouper = _create_time_grouper(df)
     if grouper is None:
-        share_susceptibles = 1 - df["immune"].mean()
+        share_susceptibles = 1 - df["immunity"].mean()
```

Comment:
This is not only a renaming of ``"immune"`` to ``"immunity"``. For the case of continuous immunity it is not directly clear how to define the share of susceptible individuals. The choice to, again, use the mean is intentionally.

Reply:
```suggestion
        share_susceptibles = 1 - (df["immunity"] > threshold).mean()
```
+ put a note in the docstring.
+ set a "reasonable" threshold.